index.js

```diff
@@ -17,11 +17,15 @@ let installed = false;
 
 let options = {
 	logger: console.error,
-	showDialog: !isDev
+	showDialog: !isDev,
+	filters: [/testError/]
```

Comment:
?

Reply:
this is a sample testError regex.
I will remove it from here and add in docs